text/0026-rest-api/rest-api.yaml

```diff
@@ -0,0 +1,2632 @@
+          parentMessageIds:
+            - 222e88a63e5aca8ef48d3b8749e2fab51d1bc7c34c5604a2933ca2f180b342c9
+            - a22137ebe61435c6d0f3e16ad148376778d7bfb36e27329f02c221ec109525a6
+            - a6db9d0b3ecb274d90c21e9dde04012b2d13ad8aa0b90e82e7d3b626be67119d
+            - fd31d9c926b5d97ae016535d66baee511dfa3b713b61d8eba2ae5c9929e38ca7
+          payload:
+            type: 2
+            index: 68656c6c6f20776f726c64
+            data: 5370616d6d696e6720646174612e0a436f756e743a203037323935320a54696d657374616d703a20323032312d30322d31315431303a32333a34392b30313a30300a54697073656c656374696f6e3a203934c2b573
+          nonce: '2102864'
+    get-messages-by-id-response-example-children:
+      value:
+        data:
+          messageId: ed3c3f1a319ff4e909cf2771d79fece0ac9bd9fd2ee49ea6c0885c9cb3b1248c
+          maxResults: 1000
+          count: 2
+          childrenMessageIds:
+            - 1c6943b0487c92fd057d4d22ad844cc37ee27fe6fbe88e5ff0d20b2233f75b9d
+            - 7ed3d67fc7b619e72e588f51fef2379e43e6e9a856635843b3f29aa3a3f1f006
+    get-messages-by-id-response-example-nochildren:
+      value:
+        data:
+          messageId: 7ed3d67fc7b619e72e588f51fef2379e43e6e9a856635843b3f29aa3a3f1f006
+          maxResults: 1000
+          count: 0
+          childrenMessageIds: []
+    get-outputs-by-id-response-example:
+      value:
+        data:
+          messageId: 9cd745ef6800c8e8c80b09174ee4b250b3c43dfa62d7c6a4e61f848febf731a0
+          transactionId: 1ee46e19f4219ee65afc10227d0ca22753f76ef32d1e922e5cbe3fbc9b5a5298
+          outputIndex: 1
+          isSpent: false
+          output:
+            type: 0
+            address:
+              type: 0
+              address: 8eaf87ac1f52eb05f2c7c0c15502df990a228838dc37bd18de9503d69afd257d
+            amount: 1000
+    get-address-balance-response-example:
+      value:
+        data:
+          addressType: 0
+          address: efdc112efe262b304bcf379b26c31bad029f616ee3ec4aa6345a366e4c9e43a3
+          balance: 1338263
+          dustAllowed: true
+    get-address-outputs-response-example:
+      value:
+        data:
+          addressType: 0
+          address: efdc112efe262b304bcf379b26c31bad029f616ee3ec4aa6345a366e4c9e43a3
+          maxResults: 1000
+          count: 4
+          outputIds:
+            - >-
+              1ee46e19f4219ee65afc10227d0ca22753f76ef32d1e922e5cbe3fbc9b5a52980100
+            - >-
+              3d36ec4afb2d634b9313f84606b98b69675a3ef6f44dcdecb18c30945b57221e0100
+            - >-
+              ee3447d088e3e2c53c5b3e56a38fdc859ca2c4b4161cf256c0462ce4d34731820100
+            - >-
+              f8bdbfb0f57ade7fbb95d31b11e2dbda9b2a35e9dc0cd3e11cb324e8a6bedc260100
+
+    get-receipts-response-example:
+      value:
+        data:
+          receipts:
+            - migratedAt: 1000
+              final: false
+              funds:
+                - tailTransactionHash: NVDARJVAYEIFDKTBFQRLI9VLTVBWCJHRYBVKWPPGJAATBPAUTE9LPZQSH9UOQEYONICHVNRAFCXA99999
```

Comment:
Should be hex encoded like `e0ce1933a900d02e0a43e2ba4e922d1a706a36a117ce1eb9ad0a74d39ee722c41b00cd227232efd76f5997e91bf8000000` for example